docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Contributing to GitOps Docs
+
+## Pre-requisites
+
+- Fork and clone the [operate-first/apps][apps-repo] repo
+- Install [JupyterBook cli][jbook-install]
+
+## Adding markdown docs
+
+All `markdown` files are found in the `docs/content` folder. Add your markdown in the appropriate subfolder.
+
+Update the `docs/_toc.yml` by adding the location of your new `markdown` to the appropriate `chapter/section`.
+
+Follow [Build the Docs section][builddocs] below ensure your markdown appears in the rendered JupyterBook.
+
+Once confirmed, commit your changes, and submit a PR to the `apps` repo.
```

Comment:
I would make "`apps` repo" a link to the apps repo (you've already defined the `apps-repo` link):

```
...PR to the [`apps` repo][apps-repo]
```